src/lib/storage/fsst_segment.cpp

```diff
@@ -0,0 +1,167 @@
+#include "fsst_segment.hpp"
+
+#include <climits>
+#include <sstream>
+#include <string>
+#include "resolve_type.hpp"
```

Comment:
System libs should come first (correct), followed by third party, followed by Hyrise headers. I am surprised that no linter bothered!?!? Or am I wrong here?